src/components/templates/Audience/Audience.tsx

```diff
@@ -251,34 +251,37 @@ export const Audience: React.FunctionComponent = () => {
   const columnsForSizedAuditorium = minColumns + auditoriumSize * 2;
 
   // 3 because 1/3 of the sie of the auditorium, * 2 because we're calculating in halves due to using cartesian coordinates + Math.abs
-  const carvedOutWidthInSeats = Math.ceil(columnsForSizedAuditorium / (3 * 2));
+  const carvedOutWidthInSeats = Math.max(
+    Math.ceil(columnsForSizedAuditorium / 5),
+    8
+  );
 
   // Keep a 16:9 ratio
-  const carvedOutHeightInSeats = Math.ceil(carvedOutWidthInSeats * (9 / 16));
+  const carvedOutHeightInSeats = Math.max(
+    Math.ceil(carvedOutWidthInSeats * (9 / 16)),
+    5
+  );
```

Comment:
Seems the comments are out of date from the code here now?

What is the `/ 5` used here for? Why does that change from what it was previously? (`3 * 2`)

It's probably worth pulling out the other `5` and the `8` into constants, as it was super confusing what they were used for. `MAX_CENTER_ROWS` / `MAX_CENTER_COLUMNS` perhaps? Or something like that?

Reply:
Fixed/Added in https://github.com/sparkletown/sparkle/pull/1237/commits/6ff3b5876858d4e306936f59a383e5f0e9c734aa